<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check_herramientas</class>
 <widget class="QDialog" name="Check_herramientas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comprobaci√≥n herramientas instaladas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pdflatex_t_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Latex instalado: herramienta PDFLATEX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="latex_label">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>260</height>
        </size>
       </property>
       <property name="text">
        <string>Check ...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="editor_latex_t_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>EDITOR DE LATEX</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="editor_text_label">
       <property name="text">
        <string>Check ...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pdfimages_t_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PDF IMAGES (poppler utils) instalado</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pdf_images_label">
       <property name="text">
        <string>Check ...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tesseract_t_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Herramienta OCR Tesseract</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="tesseract_label">
       <property name="text">
        <string>Check ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="comprobar_pushButton">
       <property name="text">
        <string>&amp;Comprobar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/check2.png</normaloff>:/iconos/graf/check2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salir_pushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salir_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Check_herramientas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
